src/PureDataNode.h

```diff
@@ -88,6 +94,7 @@ class PureDataNode : public ci::audio::Node, public pd::PdReceiver {
   void sendList(const std::string &dest, const pd::List &list);
   void sendMessage(const std::string &dest, const std::string &msg,
                    const pd::List &list = pd::List());
+  void sendNoteOn(int port, int pitch, int velocity);
```

Comment:
Curious why `port` is used here and `channel` in the `NoteOn` struct. Seems like we should pick one or the other. Channel seems like the MIDI nomenclature.